inst/include/RcppEigenWrap.h

```diff
@@ -217,6 +217,21 @@ namespace Rcpp{
             OUT get() {return OUT(vec.begin(), vec.size());}
         } ;
 
+        // Provides only Map::RowVectorX<t> export
+        template<typename T>
+        class Exporter<Eigen::Map<Eigen::Matrix<T, 1, Eigen::Dynamic> > > {
+          typedef typename Eigen::Map<Eigen::Matrix<T, 1, Eigen::Dynamic> > OUT ;
+          const static int RTYPE = ::Rcpp::traits::r_sexptype_traits<T>::rtype ;
+          Rcpp::Vector<RTYPE> vec ;
+          
+        public:
+          Exporter(SEXP x) : vec(x) {
+            if (TYPEOF(x) != RTYPE)
+              throw std::invalid_argument("Wrong R type for mapped rowvector");
+          }
+          OUT get() {return OUT(vec.begin(), vec.size());}
+        } ;
+
```

Comment:
That whole block is two-space-intended (after the initial four) but c'est la vie now, we'll get it next time or maybe I'll fix it tomorrow.